shell/store/auth.js

```diff
@@ -18,12 +18,11 @@ export const BASE_SCOPES = {
 
 const KEY = 'rc_nonce';
 
-const ERR_NONCE = 'nonce';
-
 export const LOGIN_ERRORS = {
   CLIENT:              'client',
   CLIENT_UNAUTHORIZED: 'client_unauthorized',
-  SERVER:              'server'
+  SERVER:              'server',
+  NONCE:               'nonce',
```

Comment:
wondered if we should also change this, as it's in the url. higher risk, it's not an un-factual state and would help product support though